### DISCUSSION TITLE
How can I strip trailing '^'?

### DISCUSSION THREAD

**Discussion by Coooz** [nextgenhealthcare/connect#5812] [Q&A] [answered]
Hi, 
I have exactly the same question as stated here:
https://forums.mirthproject.io/forum/mirth-connect/support/8864-deleting-empty-fields
However, the links that provide the answer are dead. 

Can anyone point me to a copy of this stripEmptyNodes template - or otherwise give a suggestion on how I get rid of trailing empty subfields?

Thanks!

**Comment by pacmano1** [comment, marked as answer]
https://github.com/nextgenhealthcare/connect-examples.

**Reply by Coooz** [reply]
Thanks @pacmano1 
Indeed exactly what I was looking for.
